tests/pos/implicitDivergenc.scala

```diff
@@ -9,5 +9,5 @@ object Test {
   implicitly[C[C[C[String]]]]
   implicitly[C[C[C[C[String]]]]]
   implicitly[C[C[C[C[C[String]]]]]]
-  implicitly[C[C[C[C[C[C[String]]]]]]] // error: no implicit argument found (because of divergence)
-}
\ No newline at end of file
+  implicitly[C[C[C[C[C[C[String]]]]]]]
```

Comment:
How come this does not diverge anymore ?

Reply:
It doesn't diverge because the reduction steps are  either neutral or reduce the size of the type being resolved. The description of the algorithm in the SIP explains in more detail.

Reply:
The reason it diverged previously is because the only check on divergence was a hard limit (via a command line argument) on the number of reduction steps permitted.